source/enforce-macos-app-location.js

```diff
@@ -30,5 +30,18 @@ module.exports = () => {
 		return;
 	}
 
-	api.app.moveToApplicationsFolder();
+	api.app.moveToApplicationsFolder({
+		conflictHandler: conflict => {
+			if (conflict === 'existsAndRunning') { // Can't replace the active version of the app
+				api.dialog.showMessageBoxSync({
+					type: 'error',
+					message: `Another version of ${appName} is currently running, quit it and retry.`,
+					buttons: [`Quit ${appName}`]
```

Comment:
This makes it ambiguous what you're "quitting". The running app or this app. I think just "OK" would be clear enough in this case.

Reply:
I think you are right 🤔 

Another option would adding an "OK" button and a "Quit & Retry" button, or something like that, that kills the other app and reopens the current one automatically. I'm not sure if that's a good idea though.

Reply:
> Another option would be adding an "OK" button and a "Quit & Retry" button, or something like that, that kills the other app and reopens the current one automatically. I'm not sure if that's a good idea though.

If one's willing to take that route maybe we may try to kill the other app in the background and retry transparently without even showing a dialog.

Reply:
I don't think quitting an app silently is a good idea. They might have important unsaved work or something.

I think just making it `OK` is the best solution. This is an edge-case anyway.